macOS_Catalina.md

```diff
@@ -17,14 +17,28 @@ If `ProductVersion` is less then `10.15` then this document is not for you. Norm
 
 ### The acid test
 To see if `Xcode Command Line Tools` is installed in a way that will work with `node-gyp`, run:
-1. `/usr/sbin/pkgutil --packages | grep CL`
-    * `com.apple.pkg.CLTools_Executables` should be listed. If it isn't, this test failed.
-2. `/usr/sbin/pkgutil --pkg-info com.apple.pkg.CLTools_Executables`
-    * `version: 11.0.0` (or later) should be listed. If it isn't, this test failed.
-    
-If both tests succeeded, _you are done_! You should be ready to install `node-gyp`.
+```
+/usr/sbin/pkgutil --pkgs |
+  grep -i com.apple.pkg.cltools |
```

Comment:
https://github.com/nodejs/node-gyp/blob/972780bde72b9a61153f3ea7ea9700cec001ffd7/gyp/pylib/gyp/xcode_emulation.py#L1436-L1438

Reply:
Which would indicate that current docs are wrong too? 😕

Reply:
@cclauss How about using this snippet:
```sh
#!/bin/bash

pkgs=(
  "com.apple.pkg.DeveloperToolsCLILeo" # standalone
  "com.apple.pkg.DeveloperToolsCLI"    # from XCode
  "com.apple.pkg.CLTools_Executables"  # Mavericks
)

for pkg in "${pkgs[@]}"; do
  output=$(/usr/sbin/pkgutil --pkg-info "$pkg" 2>/dev/null)
  if [ "$output" ]; then
    version=$(echo "$output" | grep 'version' | cut -d' ' -f2)
    break
  fi
done

if [ "$version" ]; then
  echo "Command Line Tools version: $version"
else
  echo >&2 'Command Line Tools not found'
fi
```